data_steward/tools/copy_dataset_to_output_prod.py

```diff
@@ -49,115 +55,168 @@ def get_arg_parser() -> argparse.ArgumentParser:
+                         args.src_project_id, args.src_dataset_id,
+                         args.output_prod_project_id, args.run_as_email)
+
+
+def generate_new(args):
+    """ Generate new output prod dataset, including copying fitbit dataset """
+    generate_output_prod(args.tier,
+                         args.release_tag,
+                         args.deid_stage,
+                         args.src_project_id,
+                         args.src_dataset_id,
+                         args.output_prod_project_id,
+                         args.run_as_email,
+                         copy_fitbit=True,
+                         fitbit_dataset_id=args.fitbit_dataset_id)
+
+
+def generate_output_prod(tier,
+                         release_tag,
+                         deid_stage,
+                         src_project_id,
+                         src_dataset_id,
+                         output_prod_project_id,
+                         run_as_email,
+                         copy_fitbit=True,
+                         fitbit_dataset_id=None):
     #Get credentials and instantiate client
     impersonation_creds = auth.get_impersonation_credentials(
-        args.run_as_email, SCOPES)
+        run_as_email, SCOPES)
 
-    bq_client = BigQueryClient(args.output_prod_project_id,
+    bq_client = BigQueryClient(output_prod_project_id,
                                credentials=impersonation_creds)
 
     #Create dataset with labels
-    output_dataset_name = get_dataset_name(args.tier, args.release_tag,
-                                           args.deid_stage)
-    description = f'{args.deid_stage} dataset created from {args.src_dataset_id} for {args.tier}{args.release_tag} CDR run'
+    output_dataset_name = get_dataset_name(tier, release_tag, deid_stage)
+    description = f'{deid_stage} dataset created from {src_dataset_id} for {tier}{release_tag} CDR run'
     labels = {
-        'clean': 'yes' if args.deid_stage == 'clean' else 'no',
-        'data_tier': args.tier.lower(),
-        'release_tag': args.release_tag.lower()
+        'clean': 'yes' if deid_stage == 'clean' else 'no',
+        'data_tier': tier.lower(),
+        'release_tag': release_tag.lower()
     }
 
     LOGGER.info(
-        f'Creating dataset {output_dataset_name} in {args.output_prod_project_id}...'
+        f'Creating dataset {output_dataset_name} in {output_prod_project_id}...'
     )
     dataset_object = bq_client.define_dataset(output_dataset_name, description,
                                               labels)
     bq_client.create_dataset(dataset_object, exists_ok=False)
 
-    #Copy fitbit tables to source dataset
-    LOGGER.info(
-        f'Copying fitbit tables from dataset {args.src_project_id}.{args.fitbit_dataset_id} to {args.src_project_id}.{args.src_dataset_id}...'
-    )
+    # Optionally copy fitbit tables to source dataset
+    if copy_fitbit and fitbit_dataset_id is not None:
```

Comment:
nit: You can remove `is not None`, since `str` is `True` if it is not `None` or `''(empty string)`.
```
    if copy_fitbit and fitbit_dataset_id:
```